modelforge/potential/ani.py

```diff
@@ -12,9 +15,10 @@
 
 from modelforge.utils.prop import SpeciesAEV
 
-if TYPE_CHECKING:
-    from modelforge.dataset.dataset import NNPInput
-    from .models import PairListOutputs
+
+# if TYPE_CHECKING:
+#     from modelforge.dataset.dataset import NNPInput
+#     from .models import PairListOutputs
```

Comment:
I think we can remove this

Reply:
Yes, I switched to the future imports and forgot to remove this commented out section!